__tests__/comment-tests.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../db/connection');
+const request = require('supertest');
+const testData = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const app = require('../app');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('Get article comments', () => {
+    it('Returns correct article comments', () => {
+        return request(app)
+            .get('/api/articles/6/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toEqual([{
+                    comment_id: 16,
+                    votes: 1,
+                    created_at: '2020-10-11 16:23:00',
+                    author: 'butter_bridge',
+                    body: 'This is a bad article name',
+                    article_id: 6
+                }]);
```

Comment:
Generally, if we are testing whether we are getting ALL articles of an article, I would choose an article that has multiple comments, check whether the length of the array we got back is matching up with the amount of comments the article has, and check whether the article_id is matchin within all of those comments